scripts/run.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/bash
+#SBATCH --job-name=bci
+#SBATCH --error=bci_%j_%a.err
+#SBATCH --out=bci_%j_%a.out
+#SBATCH --time=04:59:59
+#SBATCH --gres=gpu:1
+#SBATCH --cpus-per-gpu=4
+#SBATCH --constraint='GPU_SKU:A100_SXM4&GPU_MEM:80GB'
+#SBATCH --mail-type=ALL
+
+export DEEP_SSM_DATA=/scratch/groups/swl1
+module load cuda/12.4
```

Comment:
I think it was OK without these module loads (no gcc errors), etc.

Here were the errors that I did have: 

/scratch/users/xavier18/miniconda3/envs/deep_ssm/lib/python3.12/site-packages/torch/functional.py:534: UserWarning: torch.meshgrid: in an upcoming release, it will be required to pass the indexing argument. (Triggered internally at /opt/conda/conda-bld/pytorch_1729647378361/work/aten/src/ATen/native/TensorShape.cpp:3595.)
 2   return _VF.meshgrid(tensors, **kwargs)  # type: ignore[attr-defined]
 3 /scratch/users/xavier18/miniconda3/envs/deep_ssm/lib/python3.12/site-packages/lightning/fabric/plugins/environments/slurm.py:204: The `srun` command is available on your system but is not used. HINT: If your intention is to run Lightning on SLURM, prepend your python command with `srun` like so: srun python3 run.py --config-name quasi_minrnn ...
 4 GPU available: True (cuda), used: True
 5 TPU available: False, using: 0 TPU cores
 6 HPU available: False, using: 0 HPUs
 7 LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]
 8 
 9   | Name     | Type          | Params | Mode
10 ---------------------------------------------------
11 0 | model    | MinRNNDecoder | 56.2 M | train
12 1 | loss_ctc | CTCLoss       | 0      | train
13 ---------------------------------------------------
14 56.2 M    Trainable params
15 0         Non-trainable params
16 56.2 M    Total params
17 224.838   Total estimated model params size (MB)
18 20        Modules in train mode
19 0         Modules in eval mode
20 Epoch 72:  46%|████▋     | 64/138 [00:32<00:37,  1.99it/s, v_num=tnze]     
21 /scratch/users/xavier18/miniconda3/envs/deep_ssm/lib/python3.12/site-packages/lightning/pytorch/trainer/connectors/data_connector.py:424: The 'val_dataloader' does not have many workers which may be a bottleneck. Consider increasing the value of the `num_workers` argument` to `num_workers=3` in the `DataLoader` to improve performance.
22 /scratch/users/xavier18/deep_ssm/src/deep_ssm/data/data_transforms.py:176: UserWarning: Using padding='same' with even kernel lengths and odd dilation may require a zero-padded copy of the input be created (Triggered internally at /opt/conda/conda-bld/pytorch_1729647378361/work/aten/src/ATen/native/Convolution.cpp:1036.)
23   return self.conv(input, weight=self.weight, groups=self.groups, padding="same")
24 /scratch/users/xavier18/miniconda3/envs/deep_ssm/lib/python3.12/site-packages/lightning/pytorch/trainer/connectors/data_connector.py:424: The 'train_dataloader' does not have many workers which may be a bottleneck. Consider increasing the value of the `num_workers` argument` to `num_workers=3` in the `DataLoader` to improve performance.
25 Testing DataLoader 0: 100%|██████████| 14/14 [00:15<00:00,  0.91it/s]   
26 `Trainer.fit` stopped: `max_steps=10000` reached.
27 LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]
28 /scratch/users/xavier18/miniconda3/envs/deep_ssm/lib/python3.12/site-packages/lightning/pytorch/trainer/connectors/data_connector.py:424: The 'test_dataloader' does not have many workers which may be a bottleneck. Consider increasing the value of the `num_workers` argument` to `num_workers=3` in the `DataLoader` to improve performance.
29 ───────────────────────────────────────────────────────────────────────────────────────────────────────